.github/workflows/links.yml

```
@@ -0,0 +1,40 @@
+name: Links (Fail Fast)
+
+on:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 * * *"
```

```suggestion
    - cron: "0 3 1 * *"
```

@christinaausley won't be looking at this more than once a month.